NRC/yangswei_85_nrc_intensity.ipynb

```diff
@@ -0,0 +1,355 @@
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>text</th>\n",
+       "      <th>label</th>\n",
+       "      <th>label_nrc</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>RTO is the new war on the middle class don't f...</td>\n",
+       "      <td>joy</td>\n",
+       "      <td>fear</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>How do you continue with life outside of work ...</td>\n",
+       "      <td>joy</td>\n",
+       "      <td>joy</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Very desperate for a job would you know a pers...</td>\n",
+       "      <td>fear</td>\n",
+       "      <td>anticipation</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>What time do you start working most days quest...</td>\n",
+       "      <td>joy</td>\n",
+       "      <td>anticipation</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>What are good job sites to find LEGIT remote w...</td>\n",
+       "      <td>joy</td>\n",
+       "      <td>trust</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                text label     label_nrc\n",
+       "0  RTO is the new war on the middle class don't f...   joy          fear\n",
+       "1  How do you continue with life outside of work ...   joy           joy\n",
+       "2  Very desperate for a job would you know a pers...  fear  anticipation\n",
+       "3  What time do you start working most days quest...   joy  anticipation\n",
+       "4  What are good job sites to find LEGIT remote w...   joy         trust"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# predict label by NRC\n",
+    "yangswei_85_test['label_nrc'] = yangswei_85_test['text'].apply(lambda x: label_text(x, nrc_lexicon))\n",
+    "\n",
+    "yangswei_85_test.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "a3f458c0-ef02-4e30-a2d0-e7562b4fc5a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "yangswei_85_test.to_csv('yangswei_85_nrc_inten_test.csv')"
```

Comment:
here the format is 'csv' but the uploaded files are in xls. why is this discrepancy? it's better we use csv.